common/terms.html

```diff
@@ -226,7 +226,7 @@
     A <a>list object</a> is a <a>dictionary</a> that has a <code>@list</code> key.
     It may also have an <code>@index</code> key, but no other members.</dd>
   <dt><dfn>literal</dfn></dt><dd>
-    An <a>object</a> expressed as a value such as a string, number or in expanded form.</dd>
+    An <a>object</a> expressed as a value such as a string, number or in expanded form as a <a>value object</a>.</dd>
```

Comment:
I believe the punctuation is still misleading, making "or" sound like it is part of the "string, number" enumeration. Proposed change
```suggestion
    An <a>object</a> expressed as a value such as a string or number, or in expanded form as a <a>value object</a>.</dd>
```